gpflow/config/__config__.py

```diff
@@ -23,7 +30,8 @@ class _Values(enum.Enum):
     will be `GPFLOW_SUMMARY_FMT`."""
     INT = np.int32
     FLOAT = np.float64
-    POSITIVE_MINIMUM = None
+    POSITIVE_BIJECTOR = "softplus"
+    POSITIVE_MINIMUM = 1e-10
```

Comment:
I would recommend to keep same as before `1e-6`. @frgsimpson can change it to 1e-10, when he needs it :)

Reply:
@awav  I'd really advise against it. For example, think of the consequences of imposing the positive transform on likelihood.variance (which I guess is the default). A 1e-6 minimum means that no GPflow model can reach an absolute precision of better than 1e-3!

Reply:
I feel like erring on the side of having a lower default makes sense...but up to you guys.

Reply:
Okay, I think we need to revert it back to the None value and think about the documentation for this in next PRs.

Reply:
At least we know it doesn't break anything now.